tuf/api/metadata.py

```diff
@@ -463,22 +463,64 @@ def remove_key(self, role: str, keyid: str) -> None:
             del self.keys[keyid]
 
 
+class MetadataInfo:
+    """A container with information about a particular metadata file.
+    Instances of MetadataInfo are used as values in a dictionary called
+    "meta" in Timestamp and Snapshot.
+    Attributes:
+        version: An integer indicating the version of the metadata file.
+        length: An optional integer indicating the length of the metadata file.
+        hashes: An optional dictionary mapping hash algorithms to the
+            hashes resulting from applying them over the metadata file
+            contents.::
+
+                'hashes': {
+                    '<HASH ALGO 1>': '<METADATA FILE HASH 1>',
+                    '<HASH ALGO 2>': '<METADATA FILE HASH 2>',
+                    ...
+                }
+
+    """
+
+    def __init__(
+        self,
+        version: int,
+        length: Optional[int] = None,
+        hashes: Optional[Dict] = None,
+    ) -> None:
+        self.version = version
+        self.length = length
+        self.hashes = hashes
+
+    def __eq__(self, other: "MetadataInfo") -> bool:
```

Comment:
The `__eq__` implementation is needed only for the tests, right?
I guess it doesn't hurt having it :shrug: Maybe it will come handy in future.

Reply:
Yes, currently `__eq__` is needed only for tests.